providers/active_directory.go

```diff
@@ -243,9 +244,13 @@ func (s *ADProvider) getUserData(username string, password string) (goth.User, e
 		if j.Name == s.config.LDAPLastNameAttribute {
 			thisUser.LastName = j.Values[0]
 		}
+		if j.Name == s.config.LDAPGroupMembershipAttribute {
```

Comment:
@rshlin what do you think if here, instead of adding a new config field (`s.config.LDAPGroupMembershipAttribute`) we just use `s.profile.CustomUserGroupField` ?